packages/checkbox/src/checkbox-base.ts

```diff
@@ -52,11 +52,16 @@ export class CheckboxBase extends Focusable {
         this.dispatchEvent(changeEvent);
     }
 
+    protected get ariaCheckedState(): 'true' | 'false' | 'mixed' {
+        return this.checked ? 'true' : 'false';
+    }
+
     protected render(): TemplateResult {
         return html`
             <input
                 id="input"
                 type="checkbox"
+                aria-checked=${this.ariaCheckedState}
```

Comment:
unsure why you are doing this here... it would seem to be a no-op again as the checked attribute will suffice in all but the mixed case.

Reply:
I checked out the React implementation in the v3 release and they were managing it in all three states, so I thought why not. If it's not needed in those context, very much simpler. Thanks for the clarification!

Reply:
yeah - the react implementation isn't always optimal ;)